modules/imported-account/main.tf

```diff
@@ -0,0 +1,36 @@
+module "org_account_access_role" {
```

Comment:
@Gowiem the original version had a role map var which allowed various roles to be created, with various policies. TBH how you have it is probably the most common use case, and the only way I've ever used it, but might be worth making this flexible, at least for the policies?

Reply:
@joe-niland yeah, I figured keeping it to this single use-case better suited the `providers.tf` w/ `var.profile` usage because we should provision other, generic roles through a more generically named + standard `providers.tf` component than this one if we want to do that. I dunno though. Let's see what others think cause I'm fine with a different implementation thought process too.